python/arcticdb/arctic.py

```diff
@@ -34,6 +35,10 @@ class Arctic:
         InMemoryLibraryAdapter,
     ]
 
+    # For test fixture clean up
+    _created_lib_names: Optional[List[str]] = None
+    _accessed_libs: Optional[List[NativeVersionStore]] = None
```

Comment:
The interactions with this don't seem to do anything?

Reply:
Ah, the clean up was reimplemented and don't need this anymore. Will remove.

Reply:
Actually, it's still used by LMDB cleanup: https://github.com/man-group/ArcticDB/blob/1014-storage-fixtures/python/arcticdb/storage_fixtures/lmdb.py#L34,L39

Reply:
The state in library manager will suffice for it, won't it?

Reply:
No, because the `LibraryManager` don't keep track of `NativeVersionStore` instances and we need to set the `.version_store` in these instances to `None` in order to release the reference to the LMDB Storage in C++.